### PR TITLE
Globalize loopback-sdk-angular

### PR DESCRIPTION
**Pull request by davidcheung**
related to https://github.com/strongloop/loopback/issues/2422

### CODE REVIEW

**Comment by davidcheung** [comment]
@slnode test please

**Comment by davidcheung** [comment]
@slnode test please

File: .gitignore [hunk 1]
```diff
@@ -16,3 +16,6 @@ karma-xunit.xml
 checkstyle.xml
 .loopbackrc
 loopback-sdk-angular-*.tgz
+!intl/
+intl/*
+!intl/en/
```
Comment (Amir-61) [inline .gitignore]: * Add an empty line at the end of file; 
* I think you don't need if you need `!intl/`, do you?

File: apidocs/describe-builtin-models.js [hunk 2]
```diff
@@ -14,14 +15,14 @@ try {
...
-    console.log('Ignoring the error since the output file is already there.');
+    g.log('Cannot load the generator, {{node_modules}} were not installed.');
+    g.log('Ignoring the error since the output file is already there.');
     process.exit();
   }
   throw err;
 }
 
-console.log('Generating API docs for LoopBack built-in models.');
+g.log('Generating API docs for {{LoopBack}} built-in models.');
```
Comment (Amir-61) [inline apidocs/describe-builtin-models.js]: `{{API docs}}` or at least `{{API}}`; I leave the decision up to you :-)

File: package.json [hunk 3]
```diff
@@ -24,7 +24,8 @@
     "url": "https://github.com/strongloop/loopback-sdk-angular/issues"
   },
   "dependencies": {
-    "ejs": "^1.0"
+    "ejs": "^1.0",
+    "strong-globalize": "^2.5.5"
```
Comment (Amir-61) [inline package.json]: `^2.5.8`

File: .gitignore (hunk 1 above)
Comment (davidcheung) [inline .gitignore]: there is an empty line EOF, or else github gives you an icon

**Comment by 0candy** [comment]
LGTM. Please squash commits and merge.

**Comment by davidcheung** [comment]
@slnode test please

File: apidocs/describe-builtin-models.js [hunk 4]
```diff
@@ -5,6 +5,7 @@
 
 /* eslint quotes: ["error", "single"] */
 var fs = require('fs');
+var g = require('strong-globalize')();
```
Comment (rmg) [inline apidocs/describe-builtin-models.js]: This should be inside the `try`/`catch` below where `require('loopback')` is done. This is what allows `apidocs.strongloop.com` to work without installing any dependencies.

This line is the root cause of strongloop-internal/apidocs.strongloop.com#56 :-(

Comment (davidcheung) [inline apidocs/describe-builtin-models.js]: @rmg thanks for catching this!

**Pull request merged**